Documentation/Signoffs/Power_System_Camera.md

```diff
@@ -1,37 +1,125 @@
+<br />
+##  Raspberry Pi 15 W USB-C Power Supply
+The Raspberry Pi 15 W USB-C Power Supply is the power supply designed by Raspbery Pi for the Raspberry Pi 4 series. Since, it is designed for the component used it meets all the power requirements perfectly. The design team chose this option over cheaper power supplies for the reliability of this power supply. 
+<br />
+##  Raspberry Pi 5 27 W USB-C Power Supply
+This component is an exception for this system. It is the power supply required for the database system's Raspberry Pi 5 as specified in their design documentation. As opposed to creating a new request, it will be included in this section and the bill of materials for the Camera Power system. It is the power supply designed by Raspberry Pi for their Raspberry Pi 5, so with access to a working wall outlet the power supply will sufficiently provide 27 Watts at 5V via a USB-C connection. For further justification, refer to the Database system documentation.
+<br />
+
+
+
+
+
+
 
 ## BOM
 | Item     | Part Number | Quantity | Price/Unit     | Total Cost |
 | -------- | ------------| -------- |----------------|------------|
-|          |             |          |                |            |
-|          |             |          |                |            |
-|          |             |          |                |            |
-|          |             |          |                |            |
-|          |             |          |                |            |
-|Total     |             |          |                |            |
+|     DC 5 V 20 A 100 W Power Supply    |    B07Q2WQ8DS         |     1     |       $19.99         |     $19.99      |
+|     Gratury Junction Box, IP67 Waterproof Plastic Enclosure    |   B08281V2RL         |     2     |       $22.94       |     $45.88      |
+|     16 AWG Wire   |    B07FMLVF84         |     1     |       $15.88       |     $15.88      |
+|    Current Sensor Module   |   B09VL4TH23        |     2     |       $9.59       |     $19.18      |
+|     2 Way Extension Cord 15 ft |   B0BCP6H3QQ         |     1     |       $39.99       |     $39.99     |
+|    Raspberry Pi 5 27 W USB-C Power Supply   |    B0CSMBR63K        |     1     |       $15.88       |     $15.88      |
+|    Raspberry Pi 15 W USB-C Power Supply   |    B0CSMBR63K        |     1     |       $7.99      |     $7.99      |
+|Total     |             |          |                |     $164.79     |
+
+Table 3: Bill of Materials <br />
 
 ## References
-<!-- This is how to do footnotes for the references: --> 
-Here is a simple footnote[^1].
-[^1]: Reference 1
-[^2]: Reference 2 
-[^3]: Reference 3
-<!--etc.-->
+
+Parts List: <br/>
+DC 5V 20A Power Supply 5 Volt 100W AC110/230V Transformer Converter <br/>
```

Comment:
Is there a data sheet for this power supply? If not, does the amazon web page provide sufficient information to verify it is suitable?

Reply:
New power supply selected with a more robust datasheet